src/hooks/screen/useChatInfoHook.ts

```diff
@@ -22,7 +22,8 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
     alertRemoveUser,
     isAnonymousModalOpen,
     selectedUser,
-    blockModalRef
+    blockModalRef,
+    isLoadingInitChat
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new state variable `isLoadingInitChat` has been introduced and destructured from the `useGroupInfo()` hook. Ensure that this state variable is properly initialized and updated in the `useGroupInfo` hook.
```diff
-    blockModalRef
+    blockModalRef,
+    isLoadingInitChat
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->